pkgs/development/python-modules/pint/default.nix

```diff
@@ -1,21 +1,26 @@
-{ stdenv
+{ lib
 , buildPythonPackage
 , fetchPypi
+, isPy27
+, funcsigs
 }:
 
 buildPythonPackage rec {
-  pname = "pint";
+  pname = "Pint";
```

Comment:
```suggestion
  pname = "pint";
```
Put the line inside `fetchPypi` instead.

Reply:
fixed.